docs/patterns.rst

```diff
@@ -54,7 +54,7 @@ setting other values, e.g.::
 
         @property
         def LANGUAGES(self):
-            return Configuration.LANGUAGES + (('tlh', 'Klingon'),)
+            return Configuration.LANGUAGES + [('tlh', 'Klingon'),]
```

Comment:
This can be made compatible with both variants in Django:

    return list(Configuration.LANGUAGES) + [('tlh', 'Klingon'),]